pallets/capacity/src/lib.rs

```diff
@@ -562,12 +579,18 @@ impl<T: Config> Pallet<T> {
 		staker: &T::AccountId,
 		proposed_amount: BalanceOf<T>,
 	) -> BalanceOf<T> {
-		let account_balance = T::Currency::free_balance(&staker);
+		let account_balance = T::Currency::balance(&staker);
 		account_balance
 			.saturating_sub(T::MinimumTokenBalance::get())
 			.min(proposed_amount)
 	}
 
+	// Calculates the total amount of tokens that are currently unlocked for the given staker.
+	pub(crate) fn get_unlocking_total_for(staker: &T::AccountId) -> (BalanceOf<T>, Weight) {
+		let unlocks = Self::get_unstake_unlocking_for(staker).unwrap_or_default();
+		(unlock_chunks_total::<T>(&unlocks), T::DbWeight::get().reads(1))
```

Comment:
why do we return the weights here?

Reply:
Yes, this is only used for migrations.
Yes, I will work on moving it there.

As written, it returns the weight to keep track of the read.
However, as the number of reads remains constant here, the calling function could be refactored without the return.

Reply:
I think it might be a good idea to move it to the migrations file since it's not used anywhere but there. if we were to reuse it in the lib.rs we would not need to return the weights since it would be caught be the benchmarks.